Utils/Extensions/NSURLExtensions.swift

```diff
@@ -223,6 +223,10 @@ extension NSURL {
 
         return false
     }
+
+    public var isLocal: Bool {
+        return host == "localhost" || host == "127.0.0.1"
```

Comment:
> Also needs ::1 for IPv6 localhost

Nice catch!

> And do we know for sure that http://LOCALHOST/ gets turned into a lowercase localhost ?

I can't reproduce any cases where `LOCALHOST`/`LocalHost`/etc. are allowed, but I'll add a `.lowercaseString` just to be safe.